db/migrate/20230921021636_add_tweet_to_replies.rb

```diff
@@ -0,0 +1,5 @@
+class AddTweetToReplies < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :replies, :tweet, null: false, foreign_key: true
```

Comment:
same here replies already has the tweet_id column, this result in `PG::DuplicateColumn: ERROR: `